pkg/cache/cache.go

```diff
@@ -31,6 +31,8 @@ const (
 	InstanceTypesAndZonesTTL = 5 * time.Minute
 	// InstanceProfileTTL is the time before we refresh checking instance profile existence at IAM
 	InstanceProfileTTL = 15 * time.Minute
+	// ExpandedSubnetTTL is time to allow reconciliation to update the subnet data needed before expiration
```

Comment:
Is this description correct? This cache timeout is actually the time that we hold this data in the cache before we drop it. This will always be updated more frequently unless there are no node classes referencing a subnet anymore OR there is an issue with DescribeSubnets